hw4.js

```diff
@@ -0,0 +1,277 @@
+"use strict"
+
+// //task1
+
+// let i;
+// let x = prompt(i);
+// let sum = 0; 
+// function getSum(){
+//     for (let a = 0; a <= x; a++){
+        
+//         sum += a;
+//     }
+//     console.log(sum);
+// }
+// getSum();
+
+
+
+
+// // task 2
+
+// let year = 5;
+// let st = 0.17;
+// function getSum(){
+//     let i;
+//     let x = prompt(i);
+//     let sum = x * st * year;
+//     alert(sum);
+// }
+// getSum();
+
+
+// // task 3
+
+
+// function trimString (stroka, ot, doo){
+//     let x = stroka.slice(ot, doo);
+//     console.log(x);
+// }
+// trimString('asdfghjk', 2, 8);
+
+
+
+
+// // task4
+
+// function getSumNumbers(num){
+//     let x = '' + num;
+//     let sum = 0;
+//     for (let i = 0; i < x.length; i++){
+//         sum += +x[i];
+//     }
+//     console.log(sum);
```

Comment:
функция должна вернуть результа
```suggestion
//     return sum;
```